packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -7,7 +7,7 @@ import cssGridTemplateColumnsMin from '@patternfly/react-tokens/dist/esm/c_descr
 import cssHorizontalTermWidth from '@patternfly/react-tokens/dist/esm/c_description_list_m_horizontal__term_width';
 
 const cssTermWidth = {
-  name: '--pf-v5-c-description-list__term-width'
+  name: '--pf-v6-c-description-list__term-width'
```

Comment:
can we reference the object here.
e.g. `\`${styles.descriptionList}___term-width\``
that way we will not have to update it manually next major release.